cllc-public-app-test/ApiIntegrationTestBaseWithLogin.cs

```diff
@@ -21,15 +21,21 @@ public abstract class ApiIntegrationTestBaseWithLogin : IClassFixture<CustomWebA
         private readonly CustomWebApplicationFactory<Startup> _factory;
 
         public HttpClient _client { get; }
+        protected readonly IConfiguration Configuration;
 
         public ApiIntegrationTestBaseWithLogin(CustomWebApplicationFactory<Startup> fixture)
         {
+            Configuration = new ConfigurationBuilder()
+                .AddUserSecrets<Gov.Lclb.Cllb.Public.Startup>()
```

Comment:
I suggest removing this as the CustomWebApplicationFactory is already bringing in the user secrets